### PR TITLE
Front end authentication

### PR DESCRIPTION
**Pull request by howard021014**
- Restrict access to the frontend points if not logged in
- Added username and logout redirect
- Add an Alert that pops above the div
Screen Shot 2020-03-08 at 10 15 26 PM

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -25,22 +23,19 @@ class App extends React.Component {
         <div className="App">
             <Switch>
               <Route exact path="/" component={Login} />
-              <div>
+              <React.Fragment>
```
Comment (anthonynd) [inline frontend/src/App.js:26]: thats cool

File: frontend/src/BrevityNavbar.js [hunk 2]
```diff
@@ -8,7 +8,18 @@ class BrevityNavbar extends React.Component {
   constructor(props) {
     super(props);
     this.props = props;
-    this.state = { username: "Bob" };
+    this.handleLogOut = this.handleLogOut.bind(this);
+  }
+
+  handleLogOut() {
+    if (localStorage.getItem('access_token') !== null) {
```
Comment (anthonynd) [inline frontend/src/BrevityNavbar.js]: `ACCESS_TOKEN`?

**Review by anthonynd — APPROVED** [review]

File: frontend/src/BrevityNavbar.js (hunk 2 above)
Comment (howard021014) [reply frontend/src/BrevityNavbar.js]: oh oops, will update it

File: frontend/src/DocumentUploadForm.js [hunk 3]
```diff
@@ -51,7 +67,7 @@ class DocumentUploadForm extends React.Component {
               <br/>
               Start by uploading a PDF document below!
             </p>
-            <h3>Upload a PDF</h3>
+            <h3>Upload a PDF (Max file size: 80 MB)</h3>
```
Comment (anthonynd) [inline frontend/src/DocumentUploadForm.js]: Could we have it like this

Screenshot from 2020-03-09 10-19-59

File: frontend/src/DocumentUploadForm.js [hunk 4]
```diff
@@ -33,16 +34,31 @@ class DocumentUploadForm extends React.Component {
...
 
   render() {
     return (
       <div>
+        { this.state.error && 
+          <Alert variant="danger" onClose={() => { this.setState({ error: false })}} dismissible>
+            <Alert.Heading>Error</Alert.Heading>
+            <p>You have uploaded a file that is larger than 80MB, please try uploading something under that</p>
+          </Alert>
+        }
```
Comment (anthonynd) [inline frontend/src/DocumentUploadForm.js]: Could we move this `Alert` into the `Col`
Change the `<p>` to `<span>`
Change the heading string to "Upload failed"
Shorten the message string to "File is larger than 80 MB. Please try again with a smaller file."
Add this style for the alert heading
```
.alert-heading {
    font-size: 16px;
    font-weight: bold;
}
```

Should look like this in the end:
Screenshot from 2020-03-09 10-40-37

**Pull request merged**